packages/app/src/LogTable.tsx

```diff
@@ -410,7 +410,7 @@ export const RawLogTable = memo(
           size: columnSizeStorage[column] ?? 150,
         })) as ColumnDef<any>[]),
         {
-          accessorKey: 'body',
+          accessorKey: 'show-pattern',
```

Comment:
@MikeShi42 Changed accessorKey from 'body' to 'show-pattern' as it was conflicting with 'body' custom column

Reply:
accessorKey should stay `body` as it renders log's `body` property, could you provide more context on why it conflicts with custom columns?

![Screenshot 2024-05-28 at 4 45 48 PM](https://github.com/hyperdxio/hyperdx/assets/20255948/fb47a8f4-c4bc-48c9-b180-eae54e813822)

Reply:
When custom column 'body' is added it conflicts with the 'message' column which has the same accessorKey
![image](https://github.com/hyperdxio/hyperdx/assets/6557230/176904a3-8a70-4fb6-8038-727324db8c74)

This causes duplicate of body when switching to other search views in this instance:
![image](https://github.com/hyperdxio/hyperdx/assets/6557230/75ebad6d-fb58-4886-bedf-1aec7129dbbe)

Not sure if there is a different way to resolve this